env.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: appstudio.redhat.com/v1alpha1
```

Comment:
Did you mean to add this file? Or is it just something you had locally for testing purposes

Reply:
you are right, it was for local use and my bad to upload it 
will remove it